packages/eslint-plugin/src/stylex-valid-styles.js

```diff
@@ -2345,8 +2350,7 @@ const stylexValidStyles = {
 
     const legacyProps: PropLimits = {
       'grid*': { limit: null, reason: legacyReason },
-      rowGap: { limit: null, reason: legacyReason },
-      columnGap: { limit: null, reason: legacyReason },
+      gap: { limit: null, reason: 'Use `rowGap` and `columnGap` instead' },
```

Comment:
`gap` should be allowed

Reply:
will be done in a follow-up. Need to auto-expand `gap` to `row-gap` and `column-gap` to deal with out-of-order css issues internally.